dev/local-environment-dynamic/README.md

```diff
@@ -0,0 +1,135 @@
+# Dynamic Local Test Environment
+
+This document outlines the setup and operation of the dynamic local test environment for Partner Chains. This environment is designed for flexibility, allowing developers to easily configure and launch a large-scale network of Partner Chains nodes with automated setup and registration.
+
+Unlike the previous static `local-environment`, which used a fixed number of validators with pre-generated keys, this dynamic environment automates the entire lifecycle, including key generation, funding, and on-chain registration.
```

Comment:
This is two edged sword.
Recently we had discussion and we agreed that configuration for tests and other things are going to be handled by e2e-tests. So, if keys are dynamic it will require a lot of effort to use this environment for automated tests. Currently the static keys are part of e2e-tests configuration.